data_models/run_tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# script for executing Django PyUnit Tests within a Docker container.
+
+# This script should always run as if it were being called from
+# the directory it lives in.
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+
+docker build -t models_tests -f Dockerfile.tests .
+
+HOST_IP=$(ifconfig eth0 | grep "inet " | awk -F'[: ]+' '{ print $4 }')
```

Comment:
Ah cool, I'll change that. Sometimes I wish linux would follow `There should be one-- and preferably only one --obvious way to do it.` a bit better.